pages/projects/benefits-navigator/index.js

```diff
@@ -657,7 +653,7 @@ export const getStaticProps = async ({ locale }) => {
       locale: locale,
       adobeAnalyticsUrl: process.env.ADOBE_ANALYTICS_URL ?? null,
       pageData: pageData.sclabsPageV1ByPath,
-      updatesData: updatesData.sclabsPageV1List.items,
+      updatesData: pageData.sclabsPageV1ByPath.item.scLabProjectUpdates,
```

Comment:
I'm guessing we can do this because the project updates are part of the projects query? Makes sense. Can we do the same thing for the Digital Standards Playbook page?

Reply:
yessir, and i've already updated the rest of the pages with this change 👍🏾